force-app/main/triggers/classes/HOT_RequestHandler.cls

```diff
@@ -349,4 +356,17 @@ public without sharing class HOT_RequestHandler extends MyTriggers {
             }
         }
     }
+    public static void setInterpretationMethod(List<HOT_Request__c> requests) {
+        List<String> userInterpretationMethodList = new List<String>();
+        for (HOT_Request__c request : requests) {
+            userInterpretationMethodList.add(request.UserInterpretationMethod__c);
+        }
+        Map<String, Id> workTypeIdByDeveloperName = new Map<String, Id>();
+        for (WorkType workType : [SELECT Id, HOT_DeveloperName__c FROM WorkType WHERE HOT_DeveloperName__c LIKE :userInterpretationMethodList]) {
```

Comment:
Hva skjer her om userInterpretationMethodList inneholder en tom string?
Og om det eksisterer en WorkType hvor HOT_DeveloperName__c == '' ?
Kanskje legge inn en sjekk for det også?

Reply:
userIntepretationMethodList vil aldri være tom, siden jeg sjekker `request.UserInterpretationMethod__c != null` før metoden kalles på